src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/Example5.java

```diff
@@ -5,25 +5,25 @@
       <property name="format" value="^[a-z]([a-z0-9][a-zA-Z0-9]*)?$"/>
       <property name="accessModifiers" value="protected, package, private"/>
       <message key="name.invalidPattern"
-        value="Parameter name ''{0}'' must match pattern ''{1}''"/>
+        value="Name ''{0}'' must match pattern ''{1}''."/>
     </module>
     <module name="ParameterName">
       <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9]*$"/>
       <property name="accessModifiers" value="public"/>
       <message key="name.invalidPattern"
-        value="Parameter name ''{0}'' must match pattern ''{1}''"/>
+        value="Name ''{0}'' must match pattern ''{1}''."/>
     </module>
   </module>
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.naming.parametername;
+
 // xdoc section -- start
-class MyClass {
-  void method1(int v1) {} // OK
-  protected method2(int V2) {} // violation, Parameter name 'V2'
-                               // must match pattern '^[a-z]([a-z0-9][a-zA-Z0-9]*)?$'
-  private method3(int a) {} // OK
-  public method4(int b) {} // violation, Parameter name 'b'
-                           // must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'
+class Example5 {
+  void method1(int v1) {}
+  protected void method2(int V) {} // violation
```

Comment:
in this specific case when message is defined by user in confg, please keep trailing comment and make it like: 
`// violation "Parameter name 'V2' must match pattern"`